tests/test_database.py

```diff
@@ -12,15 +13,28 @@ def example_database(tmp_path:Path) -> Database:
     shutil.copytree(Path(__file__).parent/'resources/example_db', tmp_path/"example_db")
     return Database(tmp_path/"example_db")
 
-
-
-def test_load_session_from_file(example_database : Database):
-    session = Session.from_file(
+@pytest.fixture()
```

Comment:
If i understood well the goal of the commit [Remove transducer in favor of only ID in Session](https://github.com/OpenwaterHealth/OpenLIFU-python/pull/110/commits/c50d98e85b7359fa71688cd7a67fa1b5d761ef08) i think that it would be better to move helper and session tests to separate commits and to squash helper commit with [Simplify serialization test code](https://github.com/OpenwaterHealth/OpenLIFU-python/pull/110/commits/b23ff826524e8518ced86591fe48c3e4ce6bc210)

Reply:
Hmm but the unit tests being added here are testing the code changes that are made in the very same commit. I feel it should stay like this